### PR TITLE
New practice item

### PR DESCRIPTION
**Pull request by moudgilz**

### CODE REVIEW

**Comment by moudgilz** [comment]
here in this branch some practice folder added,
beside index.js file updated as per requirements, so any time any file required to run just simply un comment that link and run

File: src/app/components/Events/events.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   }
+   buttonEventCheck() {
+      this.state.prop2 = this.state.prop2 + 1;
+      this.setState({ prop2: this.state.prop2 });
+   }
+   render() {
+      return (
+         <div>
+            <ChildComponent myPropOne={this.state.prop1} myPropTwo={this.updateStatus}
+               propEvent={this.buttonEventCheck} proEventValue={this.state.prop2} />
```
Comment (jagdishkumar17) [inline src/app/components/Events/events.jsx]: can you please add example for how we can pass parameter to buttonEventCheck function on button click

Comment (moudgilz) [reply src/app/components/Events/events.jsx]: ok

**Review by moudgilz — COMMENTED** [review]
here event is modified,
now once you type anything in textbox of child page the value will pass back to parent form using event

File: src/app/components/Events/events.jsx (hunk 1 above)
Comment (moudgilz) [reply src/app/components/Events/events.jsx]: i have taken here onChangeEvent instead onClick

Comment (moudgilz) [reply src/app/components/Events/events.jsx]: button click event with parameter has called up

**Pull request merged**